test/factoryTest.js

```diff
@@ -1,64 +1,75 @@
-const SortitionPoolFactory = artifacts.require(
-  "./contracts/SortitionPoolFactory.sol",
-)
-const SortitionPool = artifacts.require("./contracts/SortitionPool.sol")
-const StakingContractStub = artifacts.require("StakingContractStub.sol")
+const { helpers, ethers } = require("hardhat")
+const { expect } = require("chai")
+const utils = require("./utils")
 
-const { mineBlocks } = require("./mineBlocks")
-
-contract("SortitionPoolFactory", (accounts) => {
-  const seed = "0xff39d6cca87853892d2854566e883008bc"
+describe("SortitionPoolFactory", () => {
+  const seed =
+    "0xff39d6cca87853892d2854566e883008bc000000000000000000000000000000"
   const minStake = 2000
   const poolWeightDivisor = 2000
-  let factory
+  let alice
+  let bob
   let staking
-  const alice = accounts[0]
-  const bob = accounts[1]
+  let factory
 
-  before(async () => {
-    staking = await StakingContractStub.new()
-    factory = await SortitionPoolFactory.new()
+  beforeEach(async () => {
+    alice = await ethers.getSigner(0)
+    bob = await ethers.getSigner(1)
+
+    const StakingContractStub = await ethers.getContractFactory(
+      "StakingContractStub",
+    )
+    staking = await StakingContractStub.deploy()
+    await staking.deployed()
+
+    const SortitionPoolFactory = await ethers.getContractFactory(
+      "SortitionPoolFactory",
+    )
+    factory = await SortitionPoolFactory.deploy()
+    await factory.deployed()
   })
 
-  describe("createSortitionPool()", async () => {
+  describe("SortitionPoolFactory", () => {
```

Comment:
According to the convention we use in other projects, shouldn't it be `createSortitionPool`? We are in the `factoryTest.js` file and we describe `SortitionPoolFactory` at the top level.

Reply:
Yes, fixed in https://github.com/keep-network/sortition-pools/pull/127/commits/4ba45853abf00e3486568229a9bbddac8475f0bb.